btb/tuning/hyperparams/boolean.py

```diff
@@ -30,28 +30,30 @@ class BooleanHyperParam(BaseHyperParam):
 
     def _within_hyperparam_space(self, values):
         if values.dtype is not np.dtype('bool'):
-            raise ValueError('Value not within hyperparameter range.')
+            raise ValueError('Values: {} not within hyperparameter range.'.format(values))
```

Comment:
`'Values {} not within hyperparameter space'.format(values)` (remove the ":" and call it "space" instead of "range")